src/main/java/dev/j3fftw/litexpansion/Items.java

```diff
@@ -326,5 +328,9 @@
         "&7Refined Smeltery"
     );
 
+    static {
+        ADVANCED_CIRCUIT.addUnsafeEnchantment(Enchantment.getByKey(Constants.GLOW_ENCHANT), 1);
```

Comment:
Any reason this is addUnsafe despite the fact max level is 1 anyway?

Reply:
No i just copied sf adding mending to infused elytra